src/app.rs

```diff
@@ -491,3 +502,24 @@ lazy_static! {
 fn validate_number(s: String) -> Result<(), String> {
     s.parse::<usize>().map(|_|()).map_err(|err| err.to_string())
 }
+
+fn validate_max_filesize(s: String) -> Result<(), String> {
+    let re = Regex::new(r#"^(\d+)([KMG])?$"#).unwrap();
```

Comment:
I don't think you need the heredoc syntax either, since you aren't using any `"` inside the regex. `r"^([0-9]+)([KMG])?$"` will suffice.